docs/rst/introduction/tools/fastddsspy.rst

```diff
@@ -0,0 +1,28 @@
+.. _vulcanexus_fastddsspy:
+
+Fast DDS Spy
+============
+
+*Fast DDS Spy* is CLI interactive tool that allows to introspect a ROS 2 network in human readable format.
+It is possible to query the network about the Participants connected, their endpoints (Publishers and Subscriptions) and the topics they communicate in.
+It is also possible to see the user data sent through network topics in a schematic format in run time.
+
+Overview
+--------
+
+*Fast DDS Spy* is a tool that introspect or "sniffing" DDS packages in the network and maintain a local database that is accessible from a interactive CLI.
+*Fast DDS Spy* responds to user commands introduced by text and print in ``stdin`` the information requested.
+This tool has several commands to interact with, that allows to get information regarding the status of the network.
+It supports to list topics existing, list Participants, Publishers, Subscriptions and even read user data in real time in a human readable format.
+
+.. figure:: /rst/figures/intro/tools/fastddsspy/fastddsspy.png
+    :align: center
+
+*Fast DDS Spy* is easily configurable and installed with a default setup, so that DDS topics, data types and entities are automatically discovered without the need to specify the types of data.
+This is because this tool exploits the DynamicTypes functionality of `eProsima Fast DDS <https://fast-dds.docs.eprosima.com>`_, the C++ implementation of the `DDS (Data Distribution Service) Specification <https://www.omg.org/spec/DDS/About-DDS/>`_ defined by the `Object Management Group (OMG) <https://www.omg.org/>`_.
```

Comment:
`exploits` sounds so.. _negative_, almost as if it is not intended. `Leverages` might be better.